Description: do not set Qt::AA_DontUseNativeMenuBar unconditionally
 This breaks the global menus on Unity desktop.
 .
 Instead of setting it forcefully to false in QGenericUnixTheme, I would
 prefer to not set this attribute at all in Qt code, to allow application
 developers to override this attribute if needed.
 .
 This cannot be forwarded as is, and needs to be discussed with upstream
 developers first.
Author: Dmitry Shachnev <mitya57@ubuntu.com>
Forwarded: no
Last-Update: 2016-09-03

--- a/src/gui/kernel/qguiapplication.cpp
+++ b/src/gui/kernel/qguiapplication.cpp
@@ -1156,11 +1156,6 @@
 
 void QGuiApplicationPrivate::createPlatformIntegration()
 {
-    // Use the Qt menus by default. Platform plugins that
-    // want to enable a native menu implementation can clear
-    // this flag.
-    QCoreApplication::setAttribute(Qt::AA_DontUseNativeMenuBar, true);
-
 
     QHighDpiScaling::initHighDpiScaling();
 
